src/components/remote-process.ts

```diff
@@ -28,7 +28,9 @@ class ESPHomeRemoteProcess extends HTMLElement {
         }
         ${coloredConsoleStyles}
       </style>
-      <div class="log"></div>
+      <div class="log${
```

Comment:
Use the `classMap` helper from Lit.

Reply:
will do (new to lit, was not aware of this functionality)

Reply:
I had to revert the `classMap` change as this code is attaching to a shadow dom and lit directives are not supported in that... I got `class="[Object object]"` instead of the strings...

Reply:
Lit directives do work in shadow dom. Their team published the first shadow dom framework :-) 

https://lit.dev/docs/templates/directives/#classmap

Reply:
@balloob I'm not sure why but the `ESPHomeRemoteProcess` does not follow the usual lit patterns
It extends `HTMLElement` not `LitElement` and uses native `attachShadow` and `innerHTML` functions so the lit directive for the `classMap` does not work:

```typescript
    const shadowRoot = this.attachShadow({ mode: "open" });
    const classes = { log: true, "blur-secrets": ESPHomeBlurSecrets.enabled };
    shadowRoot.innerHTML = `
      <style>
        :host {
          display: flex;
        }
        ${coloredConsoleStyles}
      </style>
      <div class="${classMap(classes)}"></div>
    `;
```
produces:
```html
<div class="[object Object]">
```
which looks like this
<img width="909" alt="image" src="https://github.com/esphome/dashboard/assets/771817/bfd49275-10b1-4364-aa74-d2b3095e4330">

Whereas the current implementation with the ternary operator produces the correct result:
<img width="675" alt="image" src="https://github.com/esphome/dashboard/assets/771817/776a75d1-f426-4a6f-a4de-49ecb0431366">

Reply:
Ah right. Yeah I don't remember why.